index.js

```diff
@@ -10,13 +10,12 @@ function stringIncludes(haystack, needle) {
 
 function countLetter(haystack, needle) {
   let count = 0
+  for (let i = 0; i < haystack.length; i++) {
+    if (haystack.slice(i, i + needle.length) === needle) {
```

Comment:
Also could have used `.split()` to turn the string into an array and then the `.shift()` would have worked as expected